en/assembly/mount_and_orient_controller.md

```diff
@@ -10,7 +10,7 @@ The controller should be placed on the frame top-side up, oriented so that the a
 ![FC Orientation](../../assets/qgc/setup/sensor/fc_orientation_1.png)
```

Comment:
Does position matter, or just orientation? I.e. does it need to be near COG?

What about optical flow? That is relative to COG according to docs, but is that always correct, or does it depend on FC position?

Reply:
It ideally is close to the CG, and otherwise the EKF_IMU_POS_X/Y/Z should be set.

Reply:
This is now fixed - there is a "position" section below (I did in another PR and rebased).